app/lib/support_ticket.rb

```diff
@@ -0,0 +1,58 @@
+require "gds_zendesk/client"
+require "gds_zendesk/dummy_client"
+
+class SupportTicket
+  def self.client
```

Comment:
Do we actually need this to be a static method ? It'd be cleaner if we just initialised a client each invocation